hw02/PlayerController.cs

```diff
@@ -1,11 +1,15 @@
 // TODO: Problem 1: Write in the comments what each line of code is doing.
 // What is "using" and what is System.Collections? Please provide comments on what the next three lines mean. 
-using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+// by inlcuding “using“ with different namespaces after it, you can use those functions, classes, functions, or other organized logically by just calling their namesapce in this script, and the conputer will know what todo.
+using System.Collections; // This namesapce included lots of useful things like lists, queues, bit arrays, hash tables, and dictionaries.
+using System.Collections.Generic; // Is a Generic collection, which allow me to use things like compare<T> and Dictionary.
+using UnityEngine; // It contain core function namespaces in Unity Engine, like variable type, GameObject.
+using UnityEngine.InputSystem; // Is a namespace could help to user to use different ways like device, gestures, and touch to control the game obejct.
```

Comment:
Nicely done! Using namespaces will grant you access to all those functionalities.